README.md

```diff
@@ -1,8 +1,6 @@
 # Bulkrax
-Short description and motivation.
+Bulkrax is an import gem that provides an admin interface to create and run importers for your Hyrax applications.  
```

Comment:
What do you think about punching this up a little? Something like 

> Bulkrax is a batteries included importer for Samvera applications. It currently includes support for OAIPMH (DC and Qualified DC) and CSV out of the box. It is also designed to be extensible, allowing you to easily add new importers in to your application or to include them with other gems. Bulkrax provides a full admin interface including creating, editing, scheduling and reviewing imports.

Reply:
I think this is much better than what I had.